modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/query_processor.clj

```diff
@@ -784,6 +784,10 @@
   [_]
   (CurrentMomentForm. nil))
 
+(defmethod sql.qp/->honeysql [:bigquery-cloud-sdk :now]
+  [driver _clause]
+  (trunc :second (->temporal-type :timestamp (sql.qp/current-datetime-honeysql-form driver))))
```

Comment:
should wrap it with a type-info?

Reply:
No, see this https://github.com/metabase/metabase/blob/369cd0811cb5b885a6965a4070bd04fb7e7bd4dd/modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/query_processor.clj#L338

Because the hsql-form has type `:timestamp`, Calling `temporal-type` on a `:now` clause compiled with `sql.qp/->honeysql` will result in `:timestamp`.